src/auth/auth.module.ts

```diff
@@ -0,0 +1,27 @@
+import { Module } from '@nestjs/common';
+import { JwtModule, JwtService } from '@nestjs/jwt';
+import { PassportModule } from '@nestjs/passport';
+import { AuthService } from './auth.service';
+import { AuthController } from './auth.controller';
+import { RedisModule } from '@liaoliaots/nestjs-redis';
+import { Env } from 'src/env';
+import { JwtStrategy } from './jwt.strategy';
+import { GithubModule } from 'src/github/github.module';
+
+@Module({
+  imports: [
+    GithubModule,
+    PassportModule,
+    JwtModule.register({
+      secret: Env.JWT.AUTH_TOKEN,
+      signOptions: { expiresIn: Env.JWT.EXPIRES_IN },
+    }),
+    RedisModule.forRoot({
+      config: { host: Env.REDIS.HOST, port: Env.REDIS.PORT },
+    }),
+  ],
+  providers: [AuthService, JwtStrategy],
+  exports: [AuthService],
```

Comment:
PrimathonAICodeReviewer

Ensure consistency by importing `JwtService` if it's used within the module.
```diff
+ import { JwtService } from '@nestjs/jwt';
```


<!-- This is an auto-generated comment by OSS PrimathonAICodeReviewer -->